frontend/src/app/modules/auth/sign-up/sign-up.component.html

```diff
@@ -1,81 +1,287 @@
-    </label>
-
-    <label>
-        <input
-            type="text"
-            [(ngModel)]="newUser.lastName"
-            #LastName="ngModel"
-            matInput
-            required
-            placeholder="Last Name"
-            [ngModelOptions]="{ standalone: true }" />
-        <mat-error *ngIf="submitted && form.controls['lastName'].errors">{{ getErrorMessage('lastName') }}</mat-error>
-    </label>
-</div>
-<div class="age-sex-section">
-    <app-calendar placeholder="Date of birth"></app-calendar>
-    <select required>
-        <option>Sex</option>
-        <option>Male</option>
-        <option>Female</option>
-    </select>
-</div>
-<select required>
-    <option>Country</option>
-    <option *ngFor="let country of Countries">{{ country }}</option>
-</select>
-<select required>
-    <option>Native Language</option>
-    <option>Ukrainian</option>
-</select>
-<select required>
-    <option>Level in English</option>
-    <option *ngFor="let level of EnglishLevels">{{ level }}</option>
-</select>
-<label>
-    <input
-        type="password"
-        [(ngModel)]="password"
-        #upPassword="ngModel"
-        required
-        placeholder="Password"
-        [ngModelOptions]="{ standalone: true }" />
-    <mat-error *ngIf="submitted && form.controls['password'].errors">{{ getPasswordErrorMessage() }}</mat-error>
-</label>
-<label>
-    <input
-        type="password"
-        [(ngModel)]="confirmPassword"
-        placeholder="Repeat The Password"
-        [ngModelOptions]="{ standalone: true }" />
-    <mat-error *ngIf="submitted && form.controls['confirmPassword'].errors">{{
-        getConfirmPasswordErrorMessage()
-    }}</mat-error>
-</label>
-<button mat-button (click)="validationThenSignUp()">Create Account</button>
-<div id="has-account">
-    <span>I have an account! </span>
-    <a href="auth/sign-in">Sign In</a>
+<div class="container">
+    <div class="form-wrp">
+        <h1>Create Account</h1>
+    <form #userRegisterForm="ngForm" [formGroup]="registerForm" class="form-content" (submit)="submitForm()">
+        <div class="form-group-item">
+            <input
+                type="email"
+                formControlName="email"
+                placeholder="Email"
+                required
+                email="{{this.email}}"/>
+    
+            <div class="errors-wrp">
+                <mat-error class="error" *ngIf="this.email.errors?.['required'] 
+                && (this.email.dirty || this.userRegisterForm.submitted)">
+                    <img src="/assets/user-profile/alert-triangle.svg" alt="">
+                    Field is required
+                </mat-error>
+    
+                <mat-error class="error" *ngIf="this.email.errors?.['email'] 
+                && (this.email.dirty || this.userRegisterForm.submitted)">
+                    <img src="/assets/user-profile/alert-triangle.svg" alt="">
+                    Wrong format
+                </mat-error>
+            </div>
```

Comment:
The same for all errors here (You can create two functions, one to check if it should be shown, second - what should be shown